### ISSUE TITLE
Delete certificates on user deletion

### ISSUE THREAD

**Issue by jom** [woocommerce/sensei-certificates#297]
### Steps to Reproduce
1. As a student, complete a course in order to generate a certificate.
2. As an admin, delete that student.

### What I Expected
The deleted user's certificates to be removed.

BONUS: Provide a tool to clean up old certificates.

### What Happened Instead
The certificates were left behind.

### PHP / WordPress / Sensei Certificates / Sensei LMS version
PHP 7.4 / WP 5.9.3 / Sensei LMS Certificates 2.2.1 / Sensei LMS 4.3.0 

### Browser / OS version
n/a

### Screenshot / Video
<img width="1766" alt="Screen Shot 2022-04-29 at 5 22 27 PM" src="https://user-images.githubusercontent.com/68693/165984530-dca4dcdd-c682-4011-aedf-6c31e780c991.png">

**Comment by jinnypark** [comment]
@jom - the user in 5168737-zd-woothemes noted that they did _not_ delete the student.

**Comment by jom** [comment]
Somehow the student's user account was deleted (not by Sensei). The WordPress user record has been deleted. What was left behind was Sensei's record that references the now deleted WordPress user.

**Comment by muitsun** [comment]
Hi all. I can confirm that it's a setting in WooCommerce that's deleting user accounts after a period of inactivity. I have now changed this to not delete inactive users. So, the only bug is that a deleted user's certificates should be removed.

**Comment by muitsun** [comment]
In the meantime, is there a way to clean up the orphan certificate entries?

**Comment by mui-tsun** [comment]
Sorry for the delay in replying. Would this script be incorporated into a future update? If so, I'd rather wait for the update. If not, I'll add this myself for now.

**Comment by m1r0** [comment]
Hey @mui-tsun, no problem.

> Would this script be incorporated into a future update?

I've added it to the queue, but it's hard to say when it will go in. Right now, the focus is on the main plugin, but hopefully, we will give some love to certificates soon. cc @burtrw

**Comment by mui-tsun** [comment]
@m1r0 Totally understand. I'll add the script myself for now. Thanks for your help.

**Comment by mui-tsun** [comment]
@m1r0 Forgot to ask... would the script also clear up orphan certificates that have been left behind? Or would it only remove certificates for newly deleted users? Thanks.

**Comment by m1r0** [comment]
@mui-tsun It should clear all orphan certificates.

**Comment by mui-tsun** [comment]
@m1r0 Fantastic!

**Comment by mui-tsun** [comment]
@m1r0 Hi there again. I've just got round to installing the script. I created a test user, then deleted the same user. However, all the orphan certificates are still there. Any idea? Does the test user have to have a certificate first?

**Comment by m1r0** [comment]
Thanks for the feedback @mui-tsun! Did the script delete any certificates for the other users at all? Do you happen to remember the post status (published, draft, trashed, etc.) of the 4 reminding certificate posts?

**Comment by mui-tsun** [comment]
@m1r0 Yes, many orphan certificates were deleted. Of the remaining 4, the post status is "Published". Looks just like all the other valid certificates. Thanks.
